File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string output = "";
        while (word1 !="")
        {
            output.append(word1.substr(0,1));
            output.append(word2.substr(0,1));
            word1.erase(0,1);
            word2.erase(0,1);
        }

        output.append(word2);
        return output;
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        ans = ""
        i = 0
        for i in range(len(word1)):
            ans+=word1[i]
            if i < len(word2):
                ans+=word2[i]

        if i < len(word2):
            for j in range(i+1, len(word2)):
                ans+= word2[j]

        return ans